### PR TITLE
ADM-926-fix:[frontend] fix: add two time range to test

### PR DESCRIPTION
**Pull request by yulongcai**
## Summary
add two time range to e2e test

## Note

_Null_

### CODE REVIEW

**Review by mrcuriosity-tw — APPROVED** [review]
LGTM.

**Review by neomgb — APPROVED** [review]

File: frontend/e2e/specs/side-path/charting-unhappy-path.spec.ts [hunk 1]
```diff
@@ -10,8 +9,14 @@ test.beforeAll(async () => {
 
 test('Charting unhappy path on config and metri page', async ({ homePage, configStep, metricsStep }) => {
   const rightDateRange = {
-    startDate: format(modifiedCorrectProjectFromFile.dateRange.startDate),
-    endDate: format(modifiedCorrectProjectFromFile.dateRange.endDate),
+    startDate: format(chartStepData.rightDateRange[0].startDate),
+    endDate: format(chartStepData.rightDateRange[0].endDate),
+    number: 0,
+  };
+  const rightDateRange1 = {
```
Comment (TingyuDong) [inline frontend/e2e/specs/side-path/charting-unhappy-path.spec.ts:16]: How about `rightDateRange_first`?

**Review by TingyuDong — APPROVED** [review]

**Review by TingyuDong — APPROVED** [review]
LGTM!

File: frontend/e2e/specs/side-path/charting-unhappy-path.spec.ts (hunk 1 above)
Comment (yulongcai) [reply frontend/e2e/specs/side-path/charting-unhappy-path.spec.ts]: done

**Review by TingyuDong — APPROVED** [review]
LGTM!

**Pull request merged**